### PR TITLE
architecture's_changes

### PR DESCRIPTION
**Pull request by MrVinzi**

### CODE REVIEW

File: css/component_style.css [hunk 1]
```diff
@@ -45,7 +25,7 @@ body {
     font-weight: bold; 
     padding: 10px;
     color: #ffffff;
-}
+}
```
Comment (AntoninaZ) [inline css/component_style.css:28]: redundant space!!!!

**Comment by AntoninaZ** [comment]
redundant space!!!!!!!!!!!

File: css/component_style.css (hunk 1 above)
Comment (AntoninaZ) [inline css/component_style.css]: redundant space!!!!

**Review by AntoninaZ — CHANGES_REQUESTED** [review]

File: css/component_style.css (hunk 1 above)
Comment (MrVinzi) [reply css/component_style.css]: fixed

**Review by AntoninaZ — APPROVED** [review]

**Pull request merged**